### PR TITLE
Updated translations

### PR DESCRIPTION
**Pull request by bonkey**
Updated translations

### CODE REVIEW

File: Sources/ZalandoCommerceUI/ZalandoCommerceUI/it.lproj/Localizable.strings [hunk 1]
```diff
@@ -110,13 +81,42 @@
 
 /*when happen to have unknown?*/
```
Comment (hani-ibrahim) [inline Sources/ZalandoCommerceUI/ZalandoCommerceUI/it.lproj/Localizable.strings]: will we keep the comments?

**Comment by hani-ibrahim** [comment]
👍

File: Sources/ZalandoCommerceUI/ZalandoCommerceUI/it.lproj/Localizable.strings (hunk 1 above)
Comment (bonkey) [inline Sources/ZalandoCommerceUI/ZalandoCommerceUI/it.lproj/Localizable.strings]: I mostly don't care about them unless they don't contain my CC number ;-)

**Comment by bonkey** [comment]
👍

**Pull request merged**